cumulus/pallets/executor-registry/src/lib.rs

```diff
@@ -329,7 +329,30 @@ mod pallet {
         pub fn pause_execution(origin: OriginFor<T>) -> DispatchResult {
             let who = ensure_signed(origin)?;
 
-            Ok(())
+            Executors::<T>::try_mutate(&who, |maybe_executor_config| -> DispatchResult {
+                let executor_config = maybe_executor_config
+                    .as_mut()
+                    .ok_or(Error::<T>::NotExecutor)?;
+
+                if executor_config.is_active {
+                    if TotalActiveExecutors::<T>::get() as usize == 1 {
```

Comment:
My assumption was that minimum should prevent folks from pausing execution to maintain desired number of executors (for decentralization or whatnot). With this implementation it is possible to have just one active regardless of what the minimum is configured to, which IMO defeats the purpose of minimum in the first place.

I understand that they should have at least minimum deposit locked, but we want actively participating executors, don't we?

Reply:
We had a small discussion with Jeremiah on this. Let's make sure we have at least `MinExecutorCount*0.75` executors active at any time instead of one and we should be good to go here.

Reply:
Done.